proc_T1CPMG.py

```diff
@@ -1,137 +1,144 @@
 from pyspecdata import *
 from scipy.optimize import leastsq,minimize,basinhopping
-from hermitian_function_test import hermitian_function_test, zeroth_order_ph
+from proc_scripts import fl_mod, find_echo_center, center_echo, postproc_dict
 from sympy import symbols
-fl = figlist_var()
-mpl.rcParams['figure.figsize'] = [8.0, 6.0]
+import sympy as sp
+from proc_scripts.fitting import decay
+import matplotlib.pyplot as plt
+logger = init_logging("debug")
+fl = fl_mod()
+plt.rcParams['figure.figsize'] = [8.0, 6.0]
 rcParams["savefig.transparent"] = True
 # {{{ input parameters
-#clock_correction = 1.785
-clock_correction = 0
-nodename = 'signal'
 filter_bandwidth = 5e3
 t2 = symbols('t2')
+test_for_flat_echo = False# test for flat echo and exit
+write_h5 = False#writes the hdf5 file
+read_h5 = True #reads the completed hdf5 file 
```

Comment:
with comments, but a space both before the `#` if it's a continuation like this, as well as and always put a space between the `#` and the comment